### DISCUSSION TITLE
Skeleton Module

### DISCUSSION THREAD

**Discussion by Kikamilka** [SAP/fundamental-ngx#9957] [Q&A] [answered]
Hi colleagues, 
I'm developer of SAP Minutes application, and now I'm working on some changes of our UI.
I wanted to use [Skeleton Component ](https://sap.github.io/fundamental-ngx/#/core/skeleton) but I weren't be able to import this module. Seems like `@fundamental-ngx/core` doesn't have such module (I checked in `node_modules`).
We use this version of fundamental: `"@fundamental-ngx/core": "0.36.2"`.

Could you help me please how it's correct to use/import this module?

For any details feel free contact with me directly: kamila.zaeva@sap.com

**Comment by N1XUS** [comment, marked as answer]
Hello @Kikamilka. Skeleton feature available in fundamental-ngx starting version v0.38.1-rc.3

**Reply by Kikamilka** [reply]
Thank you for reply.

As idea, maybe would be great to add this information to pages with description of components: https://sap.github.io/fundamental-ngx/#/core/skeleton.
